supporting_content/metrics/pricingquantity.md

```diff
@@ -0,0 +1,93 @@
+# Column: PricingQuantity
+
+## Example treatment for non-metered charges (e.g. Tax Treatment)
```

Comment:
Why are these examples here?  I'm sure they merit inclusion, but I don't immediately follow, and I would just want to see a bit of verbiage explaining what's happening with these Examples.  

Also, what does "treatment" mean?  That term isn't mentioned in the spec portion.  A quick Google does indeed suggest that "tax treatment" is a concept, but I've spent a ton of time with financial data and I've never seen it before.  So, if not defined here, maybe worth inclusion in the glossary for us dummies?

Reply:
```suggestion
## Example PricingQuantity values for charges where ChargeCategory is "Tax"
```
The section sub-title and content is a little jarring - for a while I thought it was a bad merge artifact of content from ChargeCategory (which conveniently has no examples for "Tax)
This looks more like discussion/scratch space content, which was used to derive the acceptable value for `PricingQuantity` when `ChargeType == Tax`. 

Suggestion would be to move it to that section.